src/v3/features/ministry/add_service_time_modal_window/index.tsx

```diff
@@ -0,0 +1,470 @@
+              reference={dropdownWithSchoolsReference}
+              open={dropdownWithSchoolsOpen}
+              anchorElement={styledRowContainerWithCreditHours.current}
+              width={styledRowContainerWithCreditHours.current?.offsetWidth + 'px'}
+              zIndex={(theme) => theme.zIndex.drawer + 3}
+            >
+              <CustomDropdownItem
+                variant="schools"
+                label={t('tr_pioneerSchool')}
+                description={t('tr_ministryTimeHours', { ministryTime: 30 })}
+                icon={<IconSchool />}
+                checked={false}
+                callback={() => setLocalCreditHoursDurationInSeconds(hoursToSeconds(30))}
+              />
+              <CustomDropdownItem
+                variant="schools"
+                label={t('tr_SKE')}
+                description={t('tr_ministryTimeHours', { ministryTime: 80 })}
+                icon={<IconSchoolForEvangelizers />}
+                checked={false}
+                callback={() => setLocalCreditHoursDurationInSeconds(hoursToSeconds(80))}
+              />
+              <CustomDropdownItem
+                variant="schools"
+                label={t('tr_languageCourse')}
+                description={t('tr_ministryTimeHours', { ministryTime: 25 })}
+                icon={<IconLanguageCourse />}
+                checked={false}
+                callback={() => setLocalCreditHoursDurationInSeconds(hoursToSeconds(25))}
+              />
+              <CustomDropdownItem
+                variant="schools"
+                label={t('tr_personalDay')}
+                description={t('tr_ministryTimeHours', { ministryTime: 5 })}
+                icon={<IconPersonalDay />}
+                checked={false}
+                sx={{
+                  borderBottom: 'none',
+                }}
+                callback={() => setLocalCreditHoursDurationInSeconds(hoursToSeconds(5))}
+              />
+            </CustomDropdownMenu>
+
+            <Box
+              sx={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                minWidth: '168px',
+                maxWidth: '168px',
+                alignItems: 'center',
+              }}
+            >
+              <MinusButton onClick={() => decrimentCreditHoursDuration()} />
+              <TextField
+                id="standard-basic"
+                variant="standard"
+                sx={{
+                  '.MuiInputBase-root::after, .MuiInputBase-root::before': { content: 'none' },
+                  '.MuiInputBase-root': {
+                    color:
+                      convertDurationInSecondsToString(localCreditHoursDurationInSeconds) != '00:00'
+                        ? 'var(--black)'
+                        : 'var(--grey-300)',
+                  },
+                  '.MuiInput-input': {
+                    textAlign: 'center',
+                    fontWeight: '550',
+                    fontFamily: 'Inter',
```

Comment:
@FussuChalice: please can we remove this `fontFamily: 'Inter'` assignment as font is changed dynamically for some languages.